### PR TITLE
Updated analyse.json (DE+EN)

### PR DESCRIPTION
**Pull request by HelTob**
Made some text changes ("Transmitted and registered test results" + "Warnings received by data donors")

### CODE REVIEW

File: src/data/analyse_en.json [hunk 1]
```diff
@@ -124,10 +124,10 @@
...
...portunity to participate in the data donation and make an important contribution to improve the app. By voluntarily sharing selected data, users can help the Robert Koch Institute to evaluate the effectiveness of the Corona-Warn-App and improve the features and usability of the app. To be a data donor, the switch in the settings under "Share data" must be set to "On".</p><p>Users who have had a brief and/or distant encounter with a person who was tested positive for corona in the past 14 days, receive a green risk warning. This means that the encounter was assumed not to be above the defined threshold value. There is no need for specific action from the users concerned.</p><p>Users who have had long and/or close encounters with at least one person who tested positive for corona in the past 14 days, receive a red risk warning. This means that the encounter was assumed to be above the defined threshold. Affected users are asked to reduce personal contacts. In case of symptoms, they should consult a general practitioner, the medical on-call service, or the local health department. The general practitioner or the local health department will decide whether a test will be provided.</p>"
```
Comment (MikeMcC399) [inline src/data/analyse_en.json:130]: Escape the quotes like this:

\"Share data\" must be set to \"On\"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Syntax problem in English version.

File: src/data/analyse_en.json (hunk 1 above)
Comment (HelTob) [reply src/data/analyse_en.json]: @MikeMcC399 
Thank you very much for your suggestion. 
I've escaped the quotes as you described it.
As editor I am using Notepad++ 

Best regards
Tobias

**Review by MikeMcC399 — APPROVED** [review]
LGTM!
The changes now allow the pages to build. 👍🏻 
I looked at the text and it explains everything.

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for reviewing and assisting here!

**Pull request merged**